go.mod

```diff
@@ -67,7 +67,7 @@ require (
 	github.com/DataDog/ebpf-manager v0.0.0-20220223090948-29871766cdd0
 	github.com/DataDog/gohai v0.0.0-20220121142033-4006c7a9c63d
 	github.com/DataDog/gopsutil v0.0.0-20220308095538-d086941833e3
-	github.com/DataDog/nikos v1.7.4
+	github.com/DataDog/nikos v1.7.5-0.20220301203122-e057b84453ad
```

Comment:
Is this related to the functional tests? If so, should this have a changelog note?